### DISCUSSION TITLE
Registration Failure

### DISCUSSION THREAD

**Discussion by Impulsleistung** [community/community#13615] [Copilot Conversations]
[ERROR] [default] [2022-03-27T08:00:38.830Z] GitHub Copilot could not connect to server. Extension activation failed: "Timed out waiting for authentication provider to register"

Environment:
Version: 1.65.2 (user setup)
Commit: c722ca6c7eed3d7987c0d5c3df5c45f6b15e77d1
Datum: 2022-03-10T14:33:55.248Z
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
Betriebssystem: Windows_NT x64 10.0.22000

**Comment by LCristinaM** [comment]
I have the same error :( , Were you able to fix it somehow?
